production/helm/loki/templates/networkpolicy.yaml

```diff
@@ -11,13 +11,15 @@ metadata:
 spec:
   podSelector:
     matchLabels:
+      name: loki
```

Comment:
Can there be 2 different values under matchLabels with same name?

Reply:
interesting catch it probably takes the first one, I will work on that part.

Reply:
will look into it for another PR